packages/grid/_modules_/grid/models/colDef/gridNumericColDef.ts

```diff
@@ -10,7 +10,7 @@ export const GRID_NUMERIC_COL_DEF: GridColTypeDef = {
   align: 'right',
   headerAlign: 'right',
   sortComparator: gridNumberComparer,
-  valueParser: (value) => Number(value),
+  valueParser: (value) => (value === '' ? null : Number(value)),
```

Comment:
```suggestion
  valueParser: (value) => Number.isNaN(value) ? null : Number(value),
```

Would this be more flexible ?

Reply:
I don't think so. An empty string becomes 0.

```js
Number.isNaN('') ? null : Number('') // 0
```

Reply:
... `''` is a valid number for JavaScript I forgot ^^